reference/sectionC_interpretation.rst

```diff
@@ -313,19 +327,27 @@ Component reference
    attribute on an :code:`import component` element.
 
 #. A component reference which is identical to the :code:`name` attribute on
-   a :code:`component` element SHALL be treated as a reference to that
-   :code:`component` element.
+   a :code:`component` element in the same infoset SHALL be treated as a
+   reference to that :code:`component` element.
 
 #. A component reference which is identical to the :code:`name` attribute on
    an :code:`import component` element SHALL be treated for the purposes of
    referencing as if the component reference appeared in the imported
    model, and referred to element with the :code:`name` specified in the
    :code:`component_ref` attribute of the :code:`import component` element.
 
-#. It is noted, for the avoidance of doubt, that CellML models MAY apply
-   the previous rule recursively, to reference an :code:`import component`
-   element which in turn references another :code:`import component`
-   element.
+#. It is noted, for the avoidance of doubt, that importing a CellML
+   :code:`component` element SHALL additionally and recursively import:
+
+      #. Any and all encapsulated :code:`component` children of the imported
+         :code:`component`, and
+      #. Any and all :code:`units` elements referenced by the imported
```

Comment:
Still needs to make clear that:
 - if I import component X, which uses units Y, that does _not_ mean I can now refer to Y in my new model
 - if I import component X, which uses units Y, and I import units Y, that _does_ let me refer to Y
 - if I import component X, which uses units Y, and I import units Y, that does not give me to Y's in my model

Reply:
Guess one way to think of it is that any `<import>` imports the entire model, and the individual component and units imports are just namespacing rules that let you refer to the imported stuff from the importing model?

Reply:
Sorry, I don't understand the last point? 

> if I import component X, which uses units Y, and I import units Y, **that does not give me to Y's in my model** 

Re namespacing stuff, yeah, kinda, but we need a way to make clear that if you import five components from the same model, you've really imported the entire model five times ...

Reply:
@MichaelClerx I think it's clearer now?  But let me know what you think (nb: added section to C1 as well as to C2, C4)